packer/linux/conf/bin/bk-configure-docker.sh

```diff
@@ -22,3 +22,8 @@ fi
 if [[ "${BUILDKITE_ENABLE_INSTANCE_STORAGE:-false}" == "true" ]] ; then
   cat <<< "$(jq '."data-root"="/mnt/ephemeral/docker"' /etc/docker/daemon.json)" > /etc/docker/daemon.json
 fi
+
+# Customise address pools
+if [[ "${DOCKER_CLASS_C_SUBNETS:-false}" == "true" ]] ; then
+  cat <<< "$(jq '."default-address-pools"=[{"base":"172.17.0.0/12","size":24},{"base":"192.168.0.0/16","size":24}]' /etc/docker/daemon.json)" > /etc/docker/daemon.json
```

Comment:
I think this is the one line we want to keep, making it default and non-optional for now.

Reply:
Hmm, it just occurred to me: often with shell pipelines it's problematic to write to the file that you're reading from. Is `jq` guaranteed to fully read `/etc/docker/daemon.json` prior to opening it for truncating-write via `>` output redirection?

Reply:
Sounds good, I'll update this PR now.
Yes, it works on our deployment and it's the same pattern as for the instance storage.